designdocs/provider_boosting_economic_model.md

```diff
@@ -0,0 +1,76 @@
+# Provider Boosting Economic Model
+
+This document outlines the economic model to be used for:
+1. determining the token value of the Reward Pool for a given Era
+2. how to calculate rewards for an individual participant in the Provider Boost program
+3. when rewards are calculated
+4. when rewards are paid out
+5. where these calculations are performed
+
+## Context and Scope:
+The Frequency Transaction Payment system uses Capacity to pay for a limited number of specific transactions on chain.  Accounts that wish to pay for transactions with Capacity must:
+1. Have an [MSA](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/accounts.md)
+2. Be a [Provider](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_registration.md) (see also [Provider Permissions and Grants](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_permissions.md))
+3. Lock up a minimum amount of FRQCY token to receive [Capacity](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/capacity.md).
+
+There is also a business case for allowing any token holder to lock up its tokens in exchange for a reward - while also targeting a Provider to receive some Capacity.
+
+
+## Problem Statement:
+A system consisting only of providers and coinless users who delegate to providers will tend toward centralization.
+To build a self-sustaining Frequency network where control is decentralized, a variety of economic solutions are needed.  One of these is the ability to lock up FRQCY token in return for something; this creates an incentive for participation and involvement with the Frequency chain fundamentals and governance.
+
+How is that so?  Capacity is how Frequency intends Providers to pay for the vast majority of their on-chain messages. In the proposed system, when Providers receive Capacity when end users lock up some FRQCY, they would lose Capacity if those end users unlock. If a Provider's Capacity from Provider Boosting is significant, this gives Provider Boosters some power over their targeted Providers. If a Provider is utilizing all or nearly all their Capacity almost every Epoch -- which they should do if trying to be economical --  then even a small percentage of lost Capacity will literally cost them to replace it.  This gives those end-users relying upon - and Boosting - their Providers the ability to exercise direct market power they did not previously have.
```

Comment:
In the proposed system, ~~when~~ Providers receive Capacity when ~~end~~ users lock up some FRQCY, they ~~would~~ lose Capacity if those ~~end~~ users unlock